launch_ros/launch_ros/actions/node.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         package: SomeSubstitutionsType,
         node_executable: SomeSubstitutionsType,
         node_name: Optional[SomeSubstitutionsType] = None,
-        node_namespace: Optional[SomeSubstitutionsType] = None,
+        node_namespace: Optional[SomeSubstitutionsType] = '',
```

Comment:
I don't follow the distinction between `''` and `None` for this argument (below it still checks for if this `is not None`). Perhaps you could either update the doc block for this argument or remove the logic considering it to be `None`? When would the user pass `None` here?

Reply:
Yeah, I agree it was unclear.
After https://github.com/ros2/launch_ros/pull/44/commits/84420b413d6c0c177399d6847fabc241cb088361, the argument doesn't accept more `None`.